### PR TITLE
Edge app for asset metadata

### PR DESCRIPTION
**Pull request by iamplugged**
image

image

Latest:

image

### CODE REVIEW

**Comment by vpetersson** [comment]
Please use the [GitHub Actions](https://github.com/Screenly/cli/#github-action) to deploy.

File: .github/workflows/deploy-asset-metadata.yml [hunk 1]
```diff
@@ -15,22 +15,19 @@ jobs:
...
-
       - name: Replace credentials
-        run: |
-          sed -i "s/GOOGLE_MAPS_API_KEY_VAL/${{ secrets.GOOGLE_MAPS_API_KEY }}/g" asset-metadata/app.yaml
-
-      - id: 'auth'
-        uses: 'google-github-actions/auth@v0'
+        run: >
+          sed -i "s/GOOGLE_MAPS_API_KEY_VAL/${{ secrets.GOOGLE_MAPS_API_KEY
+          }}/g" asset-metadata-web/app.yaml
```
[the remark this answers is not in the document]
Comment (iamplugged) [reply .github/workflows/deploy-asset-metadata.yml:21]: Linting says long line, so multilined with `>`

[the remark this answers is not in the document]
Comment (vpetersson) [reply .github/workflows/deploy-asset-metadata.yml:21]: Feel free to suppress long lines linting errors.

[the remark this answers is not in the document]
Comment (iamplugged) [reply .github/workflows/deploy-asset-metadata.yml]: Done

**Comment by iamplugged** [comment]
Removed header and footer

**Comment by vpetersson** [comment]
Closed in favor of #108.

**Pull request closed without merging**